.github/CHANGELOG.md

```diff
@@ -3,13 +3,6 @@
 <!-- Always use "1." at the start instead of "2. " or "X. " as GitHub will auto renumber everything. -->
 <!-- Use the following format below -->
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
-## Changes from 2020/10 to 2020/11
-1. [ND] Change ND Sizing - @nathaninnes (Nathan Innes)
-
-1. [GPWS] Added Pull Up, Sink Rate, Dont sink GPWS with Default Sounds & Added (Too low) terrain, flaps and gear without Sounds (Future PR by Boris) - @oliverpope03 (Oliver Pope)
-
-1. [Sounds] Added new sounds for fuel pumps, flaps, ground roll and rattles, touchdowns, and wind - @hotshotp (Boris)
-
```

Comment:
Did you mean to change the order here?

Reply:
Yes the current month was added to the bottom, since everyone kept adding changes there.